### ISSUE TITLE
Mark beta release as pre-release

### ISSUE THREAD

**Issue by strausmann** [bambulab/BambuStudio#3940]
**Is your feature request related to a problem? Please describe.**

Currently, both stable and beta releases of the BambuStudio are marked as normal versions in the GitHub releases. However, this means that it is not possible to distinguish between stable and beta releases via the GitHub API. Only the name of a release shows that it is a beta version.

**Describe the solution you'd like**

I would like every BambuStudio release to be marked with the pre-release if it is a beta version. In the same way, I would like a new stable release to be marked as "latest release".

![image](https://github.com/bambulab/BambuStudio/assets/4519228/d37594da-652b-4311-83ad-fbb69752514a)

In addition, an unstable release, i.e. a beta version, should also be tagged with the version number and the suffix "-beta".

![image](https://github.com/bambulab/BambuStudio/assets/4519228/21bafc5f-23b4-4abf-a5d7-412c5a7d47ab)

**Describe alternatives you've considered**

The solution has already been described above. This actually corresponds to a standard process for publishing software with stable and unstable versions on GitHub.

**Additional context**

**Comment by lanewei120** [comment]
thanks for the suggestion

done for 1.9.1.58 beta 4
and will be done for all the future releases

**Comment by strausmann** [comment]
Hello @lanewei120, I once drew a fork. If I want to create a release at the fork now, it looks as expected. 

![Screenshot_20240426_074218_Edge](https://github.com/bambulab/BambuStudio/assets/4519228/5b265e9d-1395-4b75-aafe-2a796ddfcd57)

 
Just adding a beta to the release tag won't make sure the GutHub API only delivers stable releases if you're looking for it. 

Please see again how you can mark the release as a prerelease. Thank you very much! 

That would improve the quality of my Chocolatey Package for the BambuStudio, in the update process. Currently, no distinction can be made between beta and stable. 

https://community.chocolatey.org/packages/bambustudio

 Thanks for the review and hopefully a better release experience.

**Comment by strausmann** [comment]
With the Orca Slicer, for example, this is done correctly. Here you can distinguish between stable and pre-release

https://github.com/SoftFever/OrcaSlicer/releases

![Screenshot_20240426_075229_Chrome](https://github.com/bambulab/BambuStudio/assets/4519228/19399b7c-94b1-499a-8a24-3c65326cb236)

**Comment by strausmann** [comment]
Is there any progress? I don't think it should be a big problem.

**Comment by lanewei120** [comment]
> This ensures that the releases at Chocolatey can also be created as a pre-package.

do you mean change the previous version?

**Comment by strausmann** [comment]
No, I mean whether we can do it in the future with pre-releases to mark them as such?

**Comment by lanewei120** [comment]
yes, we can do it in the future versions

**Comment by strausmann** [comment]
@lanewei120 

I have one more comment. Please do not simply change the assets in a GitLab release. If you have a problem with the v01.09.01.66 release on Windows, you cannot simply adapt v01.09.01.67 as assets for the v01.09.01.66 release. You don't do that. That would have been a completely new GitLab release.

**Comment by strausmann** [comment]
@walterwongbbl 

With the new release, you have unfortunately forgotten to create a tag for this release. Could you please make sure that this tag matches the version in the release title?

![image](https://github.com/bambulab/BambuStudio/assets/4519228/c3d70432-f3eb-4ce2-96b2-ccf58c1f09ca)

Without the tag and thus the version number, no Chocolatey Package can be generated automatically for Bambu Studio.

**Comment by lanewei120** [comment]
done
https://github.com/bambulab/BambuStudio/tree/v01.09.02.57

**Comment by strausmann** [comment]
> done https://github.com/bambulab/BambuStudio/tree/v01.09.02.57

Thank you :-)
